### PR TITLE
update to WildFly23

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

[droolsjbpm-build-bootstrap](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1626)
[kie-soup](https://github.com/kiegroup/kie-soup/pull/205)
[droolsjbpm-knowledge](https://github.com/kiegroup/droolsjbpm-knowledge/pull/521)
[drools](https://github.com/kiegroup/drools/pull/3628)
[optaplanner](https://github.com/kiegroup/optaplanner/pull/1333)
[appformer](https://github.com/kiegroup/appformer/pull/1143)
[jbpm](https://github.com/kiegroup/jbpm/pull/1917)
[kie-jpmml-integration](https://github.com/kiegroup/kie-jpmml-integration/pull/54)
[jbpm-wb](https://github.com/kiegroup/jbpm-wb/pull/1489)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins execute fdb

**Comment by mbiarnes** [comment]
Jenkins run fdb

**Review by mareknovotny — APPROVED** [review]

**Comment by mbiarnes** [comment]
Jenkins execute fdb

File: pom.xml [hunk 1]
```diff
@@ -73,7 +73,7 @@
     </version.com.ahome-it.lienzo.tooling.nativetools>
     <version.com.ahome-it.lienzo.tooling.common>1.0.190-RELEASE</version.com.ahome-it.lienzo.tooling.common>
     <version.com.allen-sauer.gwt.dnd>3.3.3</version.com.allen-sauer.gwt.dnd>
-    <version.com.fasterxml.jackson>2.10.4</version.com.fasterxml.jackson>
+    <version.com.fasterxml.jackson>2.12.1</version.com.fasterxml.jackson>
     <version.com.fasterxml.jackson.databind>2.10.5.1</version.com.fasterxml.jackson.databind>
```
Comment (mareknovotny) [inline pom.xml:77]: i think you forgot on updating this and line below

Comment (mareknovotny) [reply pom.xml:77]:
```suggestion
    <version.com.fasterxml.jackson.databind>${version.com.fasterxml.jackson}</version.com.fasterxml.jackson.databind>
```

File: pom.xml [hunk 2]
```diff
@@ -73,7 +73,7 @@
...
     <version.com.fasterxml.jackson.annotations>2.9.10</version.com.fasterxml.jackson.annotations>
```
Comment (mareknovotny) [inline pom.xml:78]:
```suggestion
    <version.com.fasterxml.jackson.annotations>${version.com.fasterxml.jackson}</version.com.fasterxml.jackson.annotations>
```

**Comment by mareknovotny** [comment]
I think i know why jbpm-wb fails now, the showcase uses    {version.org.jboss.errai.wildfly} and you have still different version of wildfly errai distro used as this line 
https://github.com/mbiarnes/droolsjbpm-build-bootstrap/blob/WildFly23/pom.xml#L329 
 is not changed

**Comment by mbiarnes** [comment]
Jenkins execute fdb

**Comment by mbiarnes** [comment]
Jenkins execute fdb

**Comment by mbiarnes** [comment]
Jenkins retest this

File: pom.xml [hunk 3]
```diff
@@ -270,10 +284,10 @@
     <version.org.testcontainers>1.15.2</version.org.testcontainers>
     <version.org.webjars.swagger-ui>2.2.10</version.org.webjars.swagger-ui>
     <version.org.webjars.npm.monaco-editor>0.20.0</version.org.webjars.npm.monaco-editor>
-    <version.org.wildfly>19.1.0.Final</version.org.wildfly>
-    <version.org.wildfly.arquillian>2.2.0.Final</version.org.wildfly.arquillian>
-    <version.org.wildfly.core>10.0.3.Final</version.org.wildfly.core>
-    <version.org.wildfly.common>1.5.1.Final</version.org.wildfly.common>
+    <version.org.wildfly>23.0.1.Final</version.org.wildfly>
```
Comment (mareknovotny) [inline pom.xml:287]:
```suggestion
    <version.org.wildfly>23.0.2.Final</version.org.wildfly>
```

File: pom.xml [hunk 4]
```diff
@@ -314,17 +328,17 @@
...
     <version.org.jboss.arquillian.selenium>3.13.0</version.org.jboss.arquillian.selenium>
     <version.org.jboss.arquillian.extension.drone>2.5.1</version.org.jboss.arquillian.extension.drone>
     <version.org.jboss.arquillian.graphene>2.3.2</version.org.jboss.arquillian.graphene>
     <version.com.google.guava.for-selenium>23.0</version.com.google.guava.for-selenium>
-    <version.org.jboss.errai>4.11.0.Final</version.org.jboss.errai>
+    <version.org.jboss.errai>4.12.0-SNAPSHOT</version.org.jboss.errai>
     <version.com.google.elemental2>1.1.0</version.com.google.elemental2>
     <!-- WildFly version used together with  the GWT's Super Dev Mode -->
-    <version.org.jboss.errai.wildfly>19.1.0.SP1</version.org.jboss.errai.wildfly>
+    <version.org.jboss.errai.wildfly>23.0.1-SNAPSHOT</version.org.jboss.errai.wildfly>
```
Comment (mareknovotny) [inline pom.xml:341]: we need to wait for Errai release of this Errai WF distro

File: pom.xml (hunk 1 above)
Comment (mareknovotny) [reply pom.xml]: is this your next step?

**Comment by mbiarnes** [comment]
Jenkins run fdb

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
This PR will be closed since Tiago Bento and William Siqueira are working on another PR:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1693

**Pull request closed without merging**